### PR TITLE
ECR Repository

### LINKED ISSUE #20: ECR Repository
Objectives:  
- create a custom ECR repository for hosting custom wordpress container image
  - you can use official wordpress image but from a custom Dockerfile (i.e. FROM wordpress) - not latest though - please fix a wordpress version
  - make sure to push at least the first version of it to the new ecr

Acceptance criteria:
  - I can push new container image tags to an ECR repository
  - I can pull images on ECR from EC2 instances running on my VPC
  - I can pull images on ECR from my local

### PR DESCRIPTION
**Pull request by faria-marcio**
This PR contains the ecr.tf to create the ECR repository and also update the variables.tf with the variable used by ecr.tf

### Some handy tips:

**1) Retrieve an authentication token and authenticate your Docker client to your registry.
Use the AWS CLI:**
C: aws ecr get-login-password --region ap-southeast-2 | docker login --username AWS --password-stdin aws_account_id.dkr.ecr.region.amazonaws.com/wp-image

**2) Build your Docker image using the following command. For information on building a Docker file from scratch:**
C: docker build -t wp-image ./wordpress/.

**3) After the build completes, tag your image so you can push the image to this repository:**
C: docker tag wp-image:latest aws_account_id.dkr.ecr.region.amazonaws.com/wp-image:latest

**4) Run the following command to push this image to your newly created AWS repository:**
c: docker push aws_account_id.dkr.ecr.region.amazonaws.com/wp-image:latest

### CODE REVIEW

**Review by deniseddi — DISMISSED** [review]
It looks very neat, Marcio! :) And you added the README. Nice job!

File: terraform/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+###**ECR**
```
Comment (kikobr82) [inline terraform/README.md:1]: You need to provide instructions and not links on how to do the things.. :)

**Review by kikobr82 — CHANGES_REQUESTED** [review]
documentation adjustments.

Just provide instruction(not link to instructions) on how to push and pull images from the repository.

File: terraform/README.md (hunk 1 above)
Comment (faria-marcio) [reply terraform/README.md:1]: I've updated the README.md file as per discussed with Kiko.

**Review by kikobr82 — APPROVED** [review]
lgtm

**Pull request merged**